src/tools/interop/idt/README.md

```diff
@@ -299,14 +265,58 @@ $ idt capture -h
 usage: idt capture [-h] [--platform {Android}] [--ecosystem {DemoExtEcosystem...
 ```
 
+> **IMPORTANT:** Note the following runtime expectations of ecosystems:  
+> `analyze_capture` must be async aware and not interact with stdin
+
 The platform loader functions the same as `capture/ecosystem`.
 
 For each package in `capture/platform`, the platform loader expects a module
 name matching the package name.  
 This module must contain a single class which is a subclass of
 `capture.base.PlatformLogStreamer`.
 
-Note the following runtime expectations of platforms:
-
--   Start should be able to be called repeatedly without restarting streaming.
--   Stop should not cause an error even if the stream is not running.
+## Project overview
+
+-   The entry point is in `idt.py` which contains simple CLI parsing with
+    `argparse`.
+-   `log` contains logging utilities used by everything in the project.
+
+For capture:
+
+-   `base` contains the base classes for ecosystems and platforms.
+-   `factory` contains the ecosystem and platform producer and controller
+-   `loader` is a generic class loader that dynamically imports classes
+    matching a given super class from a given directory.
+-   `utils/shell` contains a simple helper class for background and foreground
+    Bash commands.
+-   `utils/artifact` contains helper functions for managing artifacts.
+
+For discovery:
+
+-   `matter_ble` provides a simple ble scanner that shows matter devices being
+    discovered and lost, as well as their VID/PID, RSSI, etc.
+-   `matter_dnssd` provides a simple DNS-SD browser that searches for matter
+    devices and thread border routers.
```

Comment:
Does your tool do tbr discovery? Or just thread device discovery. If it can id tbrs, we should talk about a way to port that back more generally into the SDK.

Reply:
We do browse for meschop service in the DNSSD browser, but otherwise there are currently no provisions specific to TBR. As an aside, at some point I would like to have the option to run a thread capture if an appropriate RCP is available and we can join the test net. There was previously a README for this but I removed it as it was not helpful beyond a TODO.

Reply:
Hi Cecille, just wanted to flag that I've updated TXT record parsing for matter records in discovery.  
Not for TBR yet, but plan to do that too.

Reply:
[Highlighting most relevant commit](https://github.com/project-chip/connectedhomeip/pull/29931/commits/e2d3dfaaa7b4a4dd7384ac79478ec731d38cfab1)